package.json

```diff
@@ -88,8 +88,8 @@
     "cypress:open": "source .env && node scripts/run-e2e.js open",
     "prepare": "husky install",
     "version": "auto-changelog -p && git add CHANGELOG.md",
-    "storybook": "source .env && start-storybook -p 6006 -s public",
-    "build-storybook": "build-storybook -s public",
+    "storybook": "source .env && REACT_APP_ENV=test && storybook dev -p 6006",
+    "build-storybook": "REACT_APP_ENV=test && storybook build -s public",
```

Comment:
Hmm may I get context why we need this? when devs run `npm run storybook`, do we need `REACT_APP_ENV` to be test? 
also, should we leverage on the existing process.env.NODE_ENV for this?

Reply:
i discovered this by accident **after** the PR so i didn't add it to the description but will explain it here: 

1. running the worker when `REACT_APP_ENV === LOCAL_DEV` means that for some API endpoints, they **will be mocked**. this will show you wrong values on our local env, 1 example wihch i encountered was showing `@mockUser` for the avatar rather than the actual user email
2. as a result of the above, we need `REACT_APP_ENV` to be something other than `LOCAL_DEV` - i chose test because it fits storybook's purpose and we need this env to allow for `msw` to mock our endpoints (for certain stories, we retrieve data from be) 
3. re: leveraging on existing `process.env.NODE_ENV` - this doesn't exist and is instead `REACT_APP_ENV`. we have to use something other than the existing one for the reasons given above

Reply:
right ok this makes sense to me